consts/consts.go

```diff
@@ -13,3 +18,15 @@ const UserAgentMacOsChrome string = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_1
 
 const CredentialMalformedMsg string = "Empty or malformed credential data.\n비어 있거나 올바르지 않은 인증 데이터 입니다."
 const InternalError string = "An internal error occured while processing data\n데이터 처리중 내부적 오류가 발생했습니다."
+
+func IsDebug() bool {
+	debug := strings.ToLower(os.Getenv("DEBUG"))
+	return strings.Compare(debug, "true") == 0
+}
+
+func SkhusWebSite() []string {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function SkhusWebSite should have comment or be unexported](https://app.codacy.com/manual/s-owl/skhus-backend/pullRequest?prid=4955299)